templates/parachain/Dockerfile

```diff
@@ -5,6 +5,7 @@ COPY . /polkadot
 
 RUN cargo fetch
 RUN cargo build --workspace --locked --release
+RUN cargo build --features generate-readme
```

Comment:
I wouldn't add this here since it is not relevant for the scope of this Dockerfile, which is to give an entry point into `parachain-template-node`.